app/frontend/actions.js

```diff
@@ -122,10 +155,14 @@ module.exports = ({setState, getState}) => {
       const walletIsLoaded = true
       const ownAddressesWithMeta = await wallet.getFilteredVisibleAddressesWithMeta()
       const transactionHistory = await wallet.getHistory()
+      const delegationHistory = await getDelegationHistory()
```

Comment:
I guess these methods should be under `wallet`

Reply:
Moved.